src/js/types/networks/ethereum.js

```diff
@@ -36,11 +36,30 @@ export type EthereumTransaction = {
     txType?: number,
 };
 
+export type EthereumTransactionEIP1559 = {
+    to: string,
+    value: string,
+    gasLimit: string,
+    nonce: string,
+    data?: string,
+    chainId: number,
+    maxFeePerGas: string,
+    maxPriorityFeePerGas: string,
+    accessList?: EthereumAccessList[];
+};
+
+export type EthereumAccessList = {
+    address: string;
+    storageKeys: string[];
+};
```

Comment:
please add this new types to typescript definitions as well. we provide both, crazy - i know :)

https://github.com/trezor/connect/blob/develop/src/ts/types/networks/ethereum.d.ts

Reply:
Done!